lib/client-assets.php

```diff
@@ -76,6 +76,8 @@ function gutenberg_register_scripts_and_styles() {
 	gutenberg_register_vendor_scripts();
 
 	// WordPress packages.
+	wp_register_script( 'wp-tinymce', includes_url( 'js/tinymce/' ) . 'wp-tinymce.php', array( 'jquery' ) );
```

Comment:
Does anything break without the jQuery dependency?

Reply:
I don't know :) Probably not. It started as a copy/paste from my old PR

Reply:
I think it's required for some custom WordPress plugins, but we're not loading those.